lua/mta_libs/wanted_constraints.lua

```diff
@@ -92,7 +99,7 @@ local function release(ply)
     players[ply] = nil
 
     if ply.SetSuperJumpMultiplier then
-        ply:SetSuperJumpMultiplier(1.5)
+        ply:SetSuperJumpMultiplier(ply:GetInfoNum("autojump_speed_multiplier", 1.5))
```

Comment:
remove everything except this, i'll change the rest where you can't (eventually)

Reply:
👍 removing the other additions then